tests/test_atomic_structure.cc

```diff
@@ -0,0 +1,91 @@
+/**
```

Comment:
Uh damn, never crossed my mind. We completely forgot to explicitly test the atomic structure. Good that it is in now.

Reply:
some additional tests could be made up I think

Reply:
...but would be unrelated to the actual goal of this PR, which is to add gradients to the spherical expansion.